package.json

```diff
@@ -9,17 +9,23 @@
   "scripts": {
     "codegen": "graph codegen --output-dir src/types/",
     "build": "graph build",
-    "deploy-test": "graph deploy --product hosted-service abarmat/token-distribution",
-    "deploy-rinkeby": "",
-    "deploy-main": "graph deploy --product hosted-service graphprotocol/token-distribution",
+    "deploy-test": "yarn && yarn prepare:mainnet && yarn codegen && graph deploy --product hosted-service juanmardefago/dev-subgraph",
+    "deploy-goerli": "yarn && yarn prepare:goerli && yarn codegen && graph deploy --product hosted-service juanmardefago/token-distribution",
+    "deploy-arbitrum-goerli": "yarn && yarn prepare:arbitrum-goerli && yarn codegen && graph deploy --product hosted-service juanmardefago/token-distribution-l2",
+    "deploy-mainnet": "yarn && yarn prepare:mainnet && yarn codegen && graph deploy --product hosted-service graphprotocol/token-distribution",
+    "deploy-arbitrum": "yarn && yarn prepare:arbitrum && yarn codegen && graph deploy --product hosted-service graphprotocol/token-distribution-arbitrum",
+    "prepare:arbitrum-goerli": "cp subgraph_arbitrum_goerli.yaml subgraph.yaml",
+    "prepare:goerli": "cp subgraph_goerli.yaml subgraph.yaml",
+    "prepare:arbitrum": "cp subgraph_arbitrum.yaml subgraph.yaml",
+    "prepare:mainnet": "cp subgraph_mainnet.yaml subgraph.yaml",
     "lint": "yarn eslint .",
     "lint-fix": "eslint . --fix",
     "prettier": "prettier '**/*.ts'",
     "prettier-write": "prettier --write '**/*.ts'"
   },
   "devDependencies": {
-    "@graphprotocol/graph-cli": "0.29.0",
-    "@graphprotocol/graph-ts": "0.26.0",
+    "@graphprotocol/graph-cli": "0.51.2",
```

Comment:
I don't remember exactly why I needed to update it, but I think older versions weren't working anymore (running `yarn install` would fail due to some old dependencies being removed from npm)